frontend/src/pages/shipments/ShipmentOffers.tsx

```diff
@@ -35,6 +36,14 @@ const ShipmentOffers: FunctionComponent<Props> = ({ shipmentId }) => {
       {
         Header: 'ID',
         accessor: 'id',
+        Cell: ({ value }: any) => (
+          <InternalLink
+            className="font-semibold"
+            to={offerViewRoute(shipmentId, value)}
+          >
+            Offer {value}
+          </InternalLink>
+        ),
```

Comment:
Add a way to navigate to an offer's details page:
<img width="1073" alt="Screen Shot 2021-03-22 at 8 05 59 PM" src="https://user-images.githubusercontent.com/3411183/112086770-11f45c80-8b4a-11eb-82ad-027c76b5b347.png">